apps/dashboard/src/utils/schema.ts

```diff
@@ -124,7 +124,7 @@ export const buildDynamicZodSchema = (obj: JSONSchemaDto): z.AnyZodObject => {
     } else if (type === 'boolean') {
       zodValue = z.boolean(isRequired ? { message: `${capitalize(key)} is required` } : undefined);
```

Comment:
```suggestion
      zodValue = z.boolean(isRequired ? { message: `${capitalize(key)} is missing` } : undefined);
```
align with other messages